src/config.rs

```diff
@@ -1,11 +1,39 @@
 use log::*;
 use serde::Deserialize;
+use std::collections::HashMap;
 use std::error::Error;
 use std::fs;
 
+/*
+pub const RELIC_NAMES: [&str; 21] = [
```

Comment:
Note some dead code here. Could maybe be used to match to relic names in some different way.

Reply:
I may end up making these objects actual `Relic` objects mapped to the in game information. The existing ones we use are `RelicButton` so they're for UI only.